superset-frontend/plugins/plugin-chart-table/src/Styles.tsx

```diff
@@ -71,7 +71,7 @@ export default styled.div`
       cursor: pointer;
     }
     td.dt-is-filter:hover {
-      background-color: ${theme.colors.secondary.light4};
+      background-color: ${theme.colors.secondary.light4} !important;
```

Comment:
if this was working for non-integers, why do we need the important now?

Reply:
It is because table cell is using inline style when numeric cell and so I need to use `important`

Reply:
Does the inline style need to set a background color at all? If so, can we switch it to use className / Emotion?

... or the other way around, worst case.